Sources/Client.swift

```diff
@@ -45,14 +45,15 @@ open class Client {
         }
 
         self.configuration = configuration
-        self.spaceIdentifier = spaceIdentifier
+        self.spaceId = spaceId
     }
 
-    fileprivate func fetch<T: Decodable>(url: URL?, completion: @escaping (Result<T>) -> Void) -> URLSessionDataTask? {
+    // TODO: rename
```

Comment:
Hmm, so clasically I also use `fetch` for a method which returns an observable, so I guess `fetch` might be alright.

Again my swiftiness is not the best, so I'll study the guide lines a bit more ... ;)